packages/python/src/index.ts

```diff
@@ -114,10 +114,24 @@ export const build = async ({
     try {
       const json = await readFile(join(pipfileLockDir, 'Pipfile.lock'), 'utf8');
       const obj = JSON.parse(json);
-      if (!meta.isDev && obj?._meta?.requires?.python_version === '3.6') {
-        pipPath = 'pip3.6';
-        pythonPath = 'python3.6';
-        pythonRuntime = 'python3.6';
+      const version = obj?._meta?.requires?.python_version;
+      if (!meta.isDev) {
+        if (version === '3.6') {
+          pipPath = 'pip3.6';
+          pythonPath = 'python3.6';
+          pythonRuntime = 'python3.6';
+          console.warn(
+            `Warning: Python version "${version}" detected in Pipfile.lock is deprecated and will reach End-Of-Life December 2021.`
+          );
+        } else if (version === '3.9') {
+          pipPath = 'pip3.9';
+          pythonPath = 'python3.9';
+          pythonRuntime = 'python3.9';
+        } else {
+          console.warn(
+            `Warning: Invalid Python version "${version}" detected in Pipfile.lock will be ignored. http://vercel.link/python-version`
```

Comment:
Should we print which version will be used instead?

Reply:
Perhaps. We're not doing that for Node.js right now. I think we just put all other details in the docs link.

https://github.com/vercel/vercel/blob/2721b3449dd3ccd213f9bdd558e61eeba3ba710c/packages/build-utils/src/fs/node-version.ts#L57-L63

Reply:
Oh I guess for Node.js we error. Do you think we should error for Python too? I'm kind of worried about breaking existing deployments.